docs/00 - Stellar Anchor Platform.md

```diff
@@ -66,6 +80,91 @@ As you can see, the Anchor Platform receives interactions from ecosystem partici
 
 This drastically reduces the amount of code that needs to be written by the Anchor, and allows them to focus on the business logic that's specific to their businesses and use cases.
 
+## SEP-31 Flow
```

Comment:
This is awesome. 

Should we merge the [one in SEP-31 ](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#completing-a-transaction-with-a-firm-quote)with this or we would want to continue maintaining both?

Reply:
That's a great point! I also thought of using that one, but the difference between both is that this one covers the interaction between `Platform<>Anchor`, which is a concept that's not part of the SEP itself.

With that in mind, I think we should maintain both. I recognize this means more work, but it is unavoidable as far as I can see.